rust/rust.dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM janitortechnology/ubuntu-dev
```

Comment:
Nit: This is actually still `janx/ubuntu-dev`. (We should probably migrate old images under `janitortechnology/` too at some point.)